build/test.ps1

```diff
@@ -38,6 +38,8 @@ function Test-One {
     }
 }
 
+python
```

Comment:
This looks like it might be leftover from something?

Reply:
Yes, this is clearly a leftover from testing if Python was available on Windows.
```suggestion
```